src/In.ProjectEKA.HipService/Common/MaskingUtility.cs

```diff
@@ -0,0 +1,36 @@
+using System.Linq;
+using In.ProjectEKA.HipLibrary.Patient.Model;
+
+namespace In.ProjectEKA.HipService.Common
+{
+    public class MaskingUtility
```

Comment:
Codacy found an issue: [Add a 'protected' constructor or the 'static' keyword to the class declaration.](https://app.codacy.com/gh/ProjectEKA/hip-service/pullRequest?prid=5811615)